classes/Customer.php

```diff
@@ -239,7 +239,7 @@ public function add($autoDate = true, $nullValues = true)
         $this->id_shop_group = ($this->id_shop_group) ? $this->id_shop_group : Context::getContext()->shop->id_shop_group;
         $this->id_lang = ($this->id_lang) ? $this->id_lang : Context::getContext()->language->id;
         $this->birthday = (empty($this->years) ? $this->birthday : (int) $this->years . '-' . (int) $this->months . '-' . (int) $this->days);
-        $this->secure_key = md5(uniqid(rand(), true));
+        $this->secure_key = md5(uniqid(random_int(0, getrandmax()), true));
```

Comment:
random_int is available since PHP7, prefer mt_rand instead

Reply:
Thanks. Solved.